contracts/TokenSale.aes

```diff
@@ -50,8 +50,10 @@ contract TokenSale =
     , bonding_curve : BondingCurve
     , description : string }
 
-  datatype event =
-    AddVote(Vote, int)
+  datatype event
+    = AddVote(Vote, int)
+    | Buy(address, int, int)
```

Comment:
Should we include timestamp as well, or we can get it using the SDK?

Reply:
timestamp is known from the transaction/microblock